src/Worker.php

```diff
@@ -662,7 +662,7 @@ protected static function init(): void
         // Log file.
         static::$logFile ??= sprintf('%s/workerman.log', dirname(__DIR__, 2));
 
-        if (!is_file(static::$logFile)) {
+        if (!is_file(static::$logFile) && static::$logFile !== '/dev/null') {
```

Comment:
Should be better to replace is_file with file_exists function. file_exists can handle /dev/null and other special files.

Reply:
@luzrain Thanx!

I didn't want to change the semantics of the code, I only allowed /dev/null

if you insist, please make your own fix

Reply:
We need to support more options, not only `/dev/null ` like `stderr`, `syslog`, ...

I think that we need to work more with the log with error and access later:
- configure log file data to show
- have more than 1 log 
- error log only show by severity (warn, info, ...)
- ...

Reply:
maybe we introduce $noLog option as first step?

Reply:
We need to create a Log Class, to easily add more options or that a user extend the class in the way he need.

Reply:
This is very difficult task for me in general

Reply:
I know that Walkor want Workerman minimalist, and I think like him.
But a minimal Log Class, that any user could change to his needs will be a good feature.

Reply:
Actually we can do the same than nodejs.
`node my_app.js > my_app_log.log 2> my_app_err.log`

Workerman only log in daemon mode, in debug mode use STDOUT
`php start.php start > /dev/null 2> my_app_err.log`

Theorically it will work.